scripts/ci_after_init.bash

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
+ROS_DISTRO=$1
+ROS_REPO=$2
+
 apt update
 apt install -y wget dialog apt-utils psmisc
 wget https://github.com/cyberbotics/webots/releases/download/R${WEBOTS_VERSION}/webots_${WEBOTS_VERSION}_amd64.deb -O /tmp/webots.deb
 apt install -y /tmp/webots.deb xvfb
+
+if [ "${ROS_DISTRO}" = "foxy" ]; then
```

Comment:
Here we need a short comment on why this condition exists and why the TurtleBot packages are installed through apt (instead of `package.xml`)

Reply:
Done in the next commit.